go/otgrpc/options.go

```diff
@@ -17,6 +17,14 @@ func LogPayloads() Option {
 	}
 }
 
+// DoNotStartTrace returns an Option that tells the OpenTracing instrumentation to
+// only trace if the context already contains a Span; do not start a new Span
+func DoNotStartTrace() Option {
```

Comment:
can we make this a little more generic? something like

```
// Returns true if.f. a Span should be created.
type SpanFilter(parentSpanCtx opentracing.SpanContext, method string, req, resp interface{}) bool
```

... then follow the pattern for the SpanDecorator. Seems more general?